File: app/bot/handlers/private/windows.py
from contextlib import suppress

from aiogram.utils.markdown import hbold

from app.bot.manager import Manager


class Window:

    @staticmethod
    async def main_menu(manager: Manager, **_) -> None:
        """
        Display the main menu window.

        :param manager: Manager object.
        :return: None
        """
        text = manager.text_message.get("main_menu")
        with suppress(IndexError, KeyError):
            text = text.format(full_name=hbold(manager.user.full_name))
        await manager.send_message(text)
        await manager.state.set_state(None)

File: node/src/bot/handlers/private/windows.ts
import type { MyContext } from "../../../types";

/** Displays the main menu window. Mirrors Python Window.main_menu. */
export async function mainMenu(ctx: MyContext): Promise<void> {
  const text = ctx.manager.text.get("main_menu");
  await ctx.manager.sendMessage(text);
}
